src/app/services/settings.service.ts

```diff
@@ -0,0 +1,10 @@
+import { Injectable } from '@angular/core';
+
+@Injectable()
+export class SettingsService {
+
+  constructor() { }
+
+  public domain: string = 'https://api.rasterfoundry.com';
```

Comment:
We should probably define this as an env variable (like we do with webpack elsewhere) so that it fetches locally when run on a dev machine and from the correct url on staging etc